### ISSUE TITLE
Charm of life crashes mobfarm

### ISSUE THREAD

**Issue by ShaBangBinBash** [Cazadorsniper/PO3#466]
[crash-2019-07-08_21.18.00-server.txt](https://github.com/Cazadorsniper/PO3/files/3368127/crash-2019-07-08_21.18.00-server.txt)
[crash-2019-07-07_18.23.26-server.txt](https://github.com/Cazadorsniper/PO3/files/3368136/crash-2019-07-07_18.23.26-server.txt)

mobfarm is using an 11x11 expanded range mob slaughter, deleted mob slaughter, world works again.

**Comment by LordBehram** [comment]
I noticed something similar on my server. 

After upgrading to version 3.0.41 we experienced some crashes which looked random at the first sight. 
When I looked at the crash reports it was clear they weren't random at all.
In my case it wasn't with a mob slaughter. During some basic testing I noticed that my server crashed when an evolved enderman died with iron spikes.  I can see the evolved enderman taking damage and as soon as his HP reaches zero the server crashes. 

I didn't do enough testing yet, but the mod Enderman Evolution was updated to version 1.0.29 in version 3.0.41 of the modpack.

BTW noticed the same behaviour in single player.

My crash reports:
[crash-2019-07-06_08.24.49-server.txt](https://github.com/Cazadorsniper/PO3/files/3369218/crash-2019-07-06_08.24.49-server.txt)
[crash-2019-07-06_15.48.57-server.txt](https://github.com/Cazadorsniper/PO3/files/3369221/crash-2019-07-06_15.48.57-server.txt)
[crash-2019-07-06_18.17.40-server.txt](https://github.com/Cazadorsniper/PO3/files/3369223/crash-2019-07-06_18.17.40-server.txt)
[crash-2019-07-06_18.25.06-server.txt](https://github.com/Cazadorsniper/PO3/files/3369224/crash-2019-07-06_18.25.06-server.txt)

**Comment by alu-** [comment]
Same issue. Duplicate of what? Waiting on which mod update?

**Comment by Cazadorsniper** [comment]
#469

**Comment by crysyn** [comment]
I am at the start of the pack and this basically stops me from making a mob farm.. Anyone know if I can go to .40 and avoid this bug for now so I can play a bit more??

**Comment by alu-** [comment]
> 
> 
> I am at the start of the pack and this basically stops me from making a mob farm.. Anyone know if I can go to .40 and avoid this bug for now so I can play a bit more??

No clue. I disabled EndermanEvolution to "solve" this issue. IMHO that mod isn't really that important anyways.

**Comment by crysyn** [comment]
> > I am at the start of the pack and this basically stops me from making a mob farm.. Anyone know if I can go to .40 and avoid this bug for now so I can play a bit more??
> 
> No clue. I disabled EndermanEvolution to "solve" this issue. IMHO that mod isn't really that important anyways.

Thanks for the suggestion, It did allow my world to load but had the same issue with corrupting the chunk as the other solutions... Just starting another new world (on to my 4th now) this time with out EndermanEvolution mod enabled.

**Comment by mkienenb** [comment]
Also seeing the same problem (PO3-3.0.41-SERVER) once I started running the mobgrinder machine.
```
crash-2019-07-20_15.45.38-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_15.45.38-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.12.43-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.12.43-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.51.27-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.51.27-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.55.58-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
crash-2019-07-20_16.55.58-server.txt:	at p455w0rd.endermanevo.entity.EntityEvolvedEnderman.func_70097_a(EntityEvolvedEnderman.java:308)
```

**Comment by shortybsd** [comment]
Simply update Twilight Forest to version https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest/files/2756932 
Delete older version on server and client, replace with linked version. Poof no more crashes.
